src/main/java/com/google/sps/servlets/GoodDeedsServlet.java

```diff
@@ -47,26 +49,40 @@
     private static final String CONTENT_TYPE_JSON = "application/json";
     private static final String REDIRECT_HOMEPAGE = "/index.html";
     
+    /*
```

Comment:
What's the reason for the testing comment here?  Don't normally see that.

Reply:
@ShreeragJ I was taking notes during my meeting with Surbhi. Just forgot to delete them after I put them in a doc